Mathlib/Analysis/Normed/Lp/ProdLp.lean

```diff
@@ -529,6 +538,32 @@ def prodContinuousLinearEquiv : WithLp p (α × β) ≃L[𝕜] α × β where
   continuous_toFun := continuous_id
   continuous_invFun := continuous_id
 
+lemma prodContinuousLinearEquiv_apply :
```

Comment:
```suggestion
lemma coe_prodContinuousLinearEquiv :
```
since these are coercions of the equiv, rather than its application to something (the `_apply` lemma would have the rhs as `ofLp x`). Similarly for symm.